engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicProcessService.scala

```diff
@@ -93,27 +103,37 @@ class PeriodicProcessService(delegateProcessManager: ProcessManager,
           }.emptyCallback
         case Right(None) =>
           logger.info(s"No next run of $deployment. Deactivating")
-          deactivateInternal(process.processVersion.processName)
+          deactivateAction(process.processVersion.processName)
         case Left(error) =>
           // This case should not happen. It would mean periodic property was valid when scheduling a process
           // but was made invalid when rescheduling again.
           logger.error(s"Wrong periodic property, error: $error. Deactivating $deployment")
-          deactivateInternal(process.processVersion.processName)
+          deactivateAction(process.processVersion.processName)
       }
     } yield callback
   }
 
-  private def markFailed(deployment: PeriodicProcessDeployment, state: Option[ProcessState]): RepositoryAction[Unit] = {
+  def markFailed(deployment: PeriodicProcessDeployment, state: Option[ProcessState]): Future[Unit] = markFailedAction(deployment, state).run
```

Comment:
I think it no longer needs to be public?

Reply:
it is actually unsed now... I will delete it